sanic/response.py

```diff
@@ -319,9 +319,26 @@ def html(
     )
 
 
+async def validate_file(
+    request_headers: Header, last_modified: Union[datetime, float, int]
+):
+    if_modified_since = request_headers.get("If-Modified-Since")
+    if not if_modified_since:
+        return
+    if_modified_since = parsedate_to_datetime(if_modified_since)
```

Comment:
We should catch a potential `ValueError` here and respond appropriately.

Reply:
Response 400? `ValueError` only for Python 3.10, and before 3.10, it was `TypeError`. I am going to try to handle both exceptions with a 400 response.

Reply:
Or maybe just ignore the invalid value and return 200?

Reply:
Just added try except to ignore invalid If-Modified-Since header. This would be considered as solved.